docs/selinux.md

```diff
@@ -0,0 +1,65 @@
+# SELinux
+
```

Comment:
I'd like to see the page starting with a simple explanation about what is SELinux, and why someone would use it on Kubernetes. Just a few sentences/lines would be enough.

Reply:
Thanks. I just refined the document. (most of parts of "Overview" section are from Wikipedia).